### PR TITLE
homework-5

### PR DESCRIPTION
**Pull request by BBlack96**

### CODE REVIEW

File: JSBasics/Homework-5/task-2.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Створіть функцію яка приймає один параметр: age.
+ * Усередині функції перевірте, чи age більше або рівне 18.
+ * Якщо age відповідає умові, поверніть true, інакше поверніть false.
+ * Викличте вашу функцію з аргументами 25 і 15, і виведіть результат (чи є особа дорослою) в консоль для кожного
+ * випадку.
+ */
+
+const isOld = (age) => age >= 18
```
Comment (sttaran) [inline JSBasics/Homework-5/task-2.js:9]: `isAdult` would be better name :)

File: JSBasics/Homework-5/task-3.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    switch (true) {
+        case available < ordered:
+            return "Your order is too large, we don’t have enough goods."
+            break
+        case ordered === 0:
+            return "Your order is empty"
+            break
+        default:
+            return "Your order is accepted"
+            break
```
Comment (sttaran) [inline JSBasics/Homework-5/task-3.js:21]: we do not need break in default block, it is always last

**Review by sttaran — CHANGES_REQUESTED** [review]

File: JSBasics/Homework-5/task-2.js (hunk 1 above)
Comment (BBlack96) [reply JSBasics/Homework-5/task-2.js]: agree)

File: JSBasics/Homework-5/task-3.js (hunk 2 above)
Comment (BBlack96) [reply JSBasics/Homework-5/task-3.js]: sorry( my bad

**Review by sttaran — APPROVED** [review]

**Pull request merged**